starter-core/src/main/java/io/micronaut/starter/feature/ci/workflows/gcp/templates/cloudBuild.rocker.raw

```diff
@@ -0,0 +1,55 @@
+@import io.micronaut.starter.options.JdkVersion
+@import io.micronaut.starter.options.BuildTool
+
+@args (
+String projectName,
+JdkVersion jdkVersion,
+BuildTool buildTool
+)
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+    @if (buildTool.equals(BuildTool.MAVEN)) {
+
+# [START cloudbuild_maven]
+steps:
+  - name: maven:3-jdk-@jdkVersion.majorVersion()
+    entrypoint: mvn
+    args: ["test"]
+  - name: maven:3-jdk-@jdkVersion.majorVersion()
+    entrypoint: mvn
+    args: ["package", "-Dmaven.test.skip=true"]
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/@projectName", "--build-arg=JAR_FILE=target/@projectName-0.1.jar", "."]
+images: ["gcr.io/$PROJECT_ID/@projectName"]
+# [END cloudbuild_maven]
+
+    } else if (buildTool.isGradle()) {
+
+# [START cloudbuild_gradle]
+steps:
+  - name: gradle:7.4.2-jdk@jdkVersion.majorVersion()
+    entrypoint: gradle
+    args: ["test"]
+  - name: gradle:7.4.2-jdk@jdkVersion.majorVersion()
+    entrypoint: gradle
+    args: ["assemble"]
+  - name: gcr.io/cloud-builders/docker
```

Comment:
I recommend removing the step to containerize our application. 

Moreover, I think our recommended approach is to containerize the application using Gradle is to use the [Micronaut Gradle Plugin docker related tasks](https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/#_docker_support)